h2o-core/src/main/java/hex/ModelBuilder.java

```diff
@@ -1880,9 +1880,9 @@ private TwoDimTable makeCrossValidationSummaryTable(Key[] cvmodels) {
     int N = cvmodels.length;
     int extra_length=2; //mean/sigma/cv1/cv2/.../cvN
     String[] colTypes = new String[N+extra_length];
-    Arrays.fill(colTypes, "string");
+    Arrays.fill(colTypes, "float");
     String[] colFormats = new String[N+extra_length];
-    Arrays.fill(colFormats, "%s");
+    Arrays.fill(colFormats, "%f");
     String[] colNames = new String[N+extra_length];
```

Comment:
Is it possible to use integer type for column types?

Reply:
The table is filled with float numbers [here](https://github.com/h2oai/h2o-3/pull/5601/files#diff-892e6d71b6905c34c9126b46bf544491e10b6c6394a83b58de5a2ead0eca6ca7R1960), [here](https://github.com/h2oai/h2o-3/pull/5601/files#diff-892e6d71b6905c34c9126b46bf544491e10b6c6394a83b58de5a2ead0eca6ca7R1966) and [here](https://github.com/h2oai/h2o-3/pull/5601/files#diff-892e6d71b6905c34c9126b46bf544491e10b6c6394a83b58de5a2ead0eca6ca7R1975-R1976). If you take a look at the FLOW UI screenshot in PR description, the actual values have some decimal places.